CMSgov/easi-app
# [EASI-2113] Fix intake review contacts

# EASI-2113

## Changes and Description

- Fixes system intake review contacts bug - contacts on review screen weren't updating after being verified.
- Updated [src/components/SystemIntakeReview/index.tsx](https://github.com/CMSgov/easi-app/compare/main...EASI-2113/fix-intake-review-contacts#diff-e114b764c3766d01672595096885baa01080a297cca75d62589f8a8f6d54b607) to use updated contacts table instead of contacts data from system intake



## How to test this change

- Verify contact on system intake request
  - Click on system intake from homepage
  - Click 'Actions' link on left sidebar
  - Select action
  - Verify Business Owner, Product Manager, or ISSO
- Click "Intake Request" link at top of left sidebar
- Confirm that system intake review Contact Details section shows updated contact info

<img width="1130" alt="Screen Shot 2022-10-25 at 3 43 50 PM" src="https://user-images.githubusercontent.com/60187543/197878309-0861930f-02f7-4058-ba0f-db6162898384.png">

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
